builder.nix

```diff
@@ -106,28 +117,32 @@ pkgs.stdenv.mkDerivation {
     bash perl python3 dtc
   ] ++ lib.optional (!lib.versionAtLeast release "21" && release != "snapshot") python2;
 
-  buildPhase = let
-    makeArgs = lib.concatStringsSep " " [
-      "-j$NIX_BUILD_CORES"
-      "DISABLED_SERVICES='${lib.concatStringsSep " " disabledServices}'"
```

Comment:
These flags are provided automatically by the default `buildPhase`: https://github.com/NixOS/nixpkgs/blob/74753257638e0858632599ca0f239631d35bda49/pkgs/stdenv/generic/setup.sh#L1319-L1322

Reply:
These flags are provided automatically by the default `buildPhase`: https://github.com/NixOS/nixpkgs/blob/74753257638e0858632599ca0f239631d35bda49/pkgs/stdenv/generic/setup.sh#L1319-L1322